definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -108,5 +108,137 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "GeneralReferralPaymentType",
+    "PageFieldDisplayOrder": 1,
```

Comment:
I think you just need to add the field "Amount to pay: £50" here in first position. 
You can do the same as the field called "LabelSolPaymentPara-1"

Reply:
Youre right! I had completely missed this. Keep eye!